patches/server/0033-Added-list-of-ItemEntity-s-that-ignore-explosions.patch

```diff
@@ -0,0 +1,45 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Hermelijn15 <j.s.dekker-1@student.utwente.nl>
+Date: Fri, 17 Nov 2023 20:09:03 +0100
+Subject: [PATCH] Added list of ItemEntity's that ignore explosions
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/Entity.java b/src/main/java/net/minecraft/world/entity/Entity.java
+index 50259042ebafe520342bdb1a4b7e6b9138b8acbd..ce082ea8e401ac37c32f6418ecc41a51e235e1aa 100644
+--- a/src/main/java/net/minecraft/world/entity/Entity.java
++++ b/src/main/java/net/minecraft/world/entity/Entity.java
+@@ -4411,6 +4411,12 @@ public abstract class Entity implements Nameable, EntityAccess, CommandSource {
+         return InteractionResult.PASS;
+     }
+ 
++    // Sakura start - getter to get level in ItemEntity.java
```

Comment:
This isn't necessary. You can get the entity level by calling `level()` within ItemEntity.

Reply:
even when it is private? I will try tomorrow

Reply:
Only the field is private the `level()` accessor is public

Reply:
ah makes sense